src/components/ui-shell/_shell-switcher.scss

```diff
@@ -0,0 +1,160 @@
+@import '../../globals/scss/functions';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/typography';
+@import 'theme';
+@import 'functions';
+
+@mixin shell-switcher {
+  .#{$prefix}--panel--overlay {
+    position: fixed;
+    top: units(6);
+    left: unset;
```

Comment:
You may want to test this with IE11, where [`unset` is not supported](https://developer.mozilla.org/en-US/docs/Web/CSS/unset#Browser_compatibility).